pkg/skaffold/docker/parse.go

```diff
@@ -262,7 +262,7 @@ func extractCopyCommands(ctx context.Context, nodes []*parser.Node, onlyLastImag
 		case command.Env:
 			// one env command may define multiple variables
 			for node := node.Next; node != nil && node.Next != nil; node = node.Next.Next {
-				envs = append(envs, fmt.Sprintf("%s=%s", node.Value, node.Next.Value))
+				envs = append(envs, fmt.Sprintf("%s=%s", node.Value, unquote(node.Next.Value)))
```

Comment:
nit: can you add a test case in parse_test.go that fails with the original issue (eg: env vars are quoted incorrectly)

Reply:
The original issue is that skaffold fails when env vars are quoted correctly though. (It works as expected with wrongly quoted vars -> `unable to stream build output: failed to process "\"asdfsafd": unexpected end of statement while looking for matching double-quote`)

Reply:
The problem occurs when trying to verify whether that directory exists, here:
https://github.com/GoogleContainerTools/skaffold/blob/fc060164fcb42d7d648df5ed5125571fde572e3f/pkg/skaffold/docker/parse.go#L170-L173

But AFAIK, this will happen at run time so in this case I think it would make more sense adding a directory and then a COPY statement to one of the tests like this one instead: https://github.com/GoogleContainerTools/skaffold/blob/fc060164fcb42d7d648df5ed5125571fde572e3f/pkg/skaffold/initializer/testdata/init/hello/Dockerfile 

What do you think?

Reply:
Yes you are correct, sorry about my suggestion before.  If possible if you could add a test like the one you suggested that would be great to make sure that this fix works as expected and we don't break this functionality with future changes.

Reply:
Just added a test. Let me know if it's not how it should be.